src/Adapter/Alias/Repository/AliasRepository.php

```diff
@@ -129,9 +133,9 @@ public function get(AliasId $aliasId): Alias
      * @param string $alias
      * @param string $searchTerm
      *
-     * @return bool
+     * @return Alias|null
      */
-    public function aliasExists(string $alias, string $searchTerm): bool
+    public function getAliasIfExist(string $alias, string $searchTerm): ?Alias
```

Comment:
Renaming a public method is BC break, + there's a typo

```suggestion
    public function getAliasIfExists(string $alias, string $searchTerm): ?Alias
```

Reply:
For the BC break, yes and no! In fact, this function is written in 2023 by Darius when the migration of Aliases has been started. But, this migration wasn't really done and although this was behind a feature flag, this first atempt of migration was not usable at all. For this 9.0.x version, isn't a big deal to change this signature.

Reply:
ok